app/js/DocumentUpdaterManager.js

```diff
@@ -88,6 +88,15 @@ const DocumentUpdaterManager = {
     })
   },
 
+  _getPendingUpdateListKey() {
+    const shard = _.random(0, 10)
```

Comment:
I think there is an off by one error here, as the range of `_.random` is 0 to 10 inclusive: https://lodash.com/docs#random

In docupdater dispatchers are created with `_.times(10, ...)`  which has the range 0-9.

To match docupdater, real-time needs to use `_.random(0,N-1)` if the number of shards is N.

```
> _.times(10)
[
  0, 1, 2, 3, 4,
  5, 6, 7, 8, 9
]
> _.max(_.times(1000, function () { return _.random(0,10) }))
10
```